Framework/src/CheckRunner.cxx

```diff
@@ -40,6 +40,7 @@ using namespace o2::configuration;
 using namespace o2::monitoring;
 using namespace o2::quality_control::core;
 using namespace o2::quality_control::repository;
+using namespace std;
```

Comment:
I don't see where you access some STL class without `std` here. Please remove it if it is not necessary.
```suggestion
```